app/views/templates/search/channels_landing.rb

```diff
@@ -56,486 +34,480 @@ def navigation
+          news: {
+            items: news_items,
+            blogurl: 'http://blog.europeana.eu/tag/#' + @channel.id
           }
         }
       end
-      
+
 
       def navigation_common
         {
-          :global  => {
-            :options => {
-              :search_active => false,
-              :settings_active => true
+          global: {
+            options: {
+              search_active: false,
+              settings_active: true
             },
-            :logo =>{
-              :url => root_url,
-              :text => "Europeana Search"
+            logo: {
+              url: root_url,
+              text: 'Europeana Search'
             },
-            :primary_nav =>{
-              :items => [
+            primary_nav: {
+              items: [
                 {
-                  :url => root_url,
-                  :text => "Home",
-                  :is_current =>false
+                  url: root_url,
+                  text: 'Home',
+                  is_current: false
                 },
                 {
-                  :url => root_url + "channels/art",
-                  :text => "Channels",
-                  :is_current =>true,
-                  :submenu => {
-                    :items => [
+                  url: root_url + 'channels/art',
+                  text: 'Channels',
+                  is_current: true,
+                  submenu: {
+                    items: [
                       {
-                        :url  => root_url + "channels/art",
-                        :text => "Art History"
+                        url: root_url + 'channels/art',
+                        text: 'Art History'
                       },
                       {
-                        :url  => root_url + "channels/music ",
-                        :text => "Music"
+                        url: root_url + 'channels/music',
+                        text: 'Music'
                       }
                     ]
                   }
                 },
                 {
-                  :url => "http://exhibitions.europeana.eu/",
-                  :text => "Exhibitions"
+                  url: 'http://exhibitions.europeana.eu/',
+                  text: 'Exhibitions'
                 },
                 {
-                  :url => "http://blog.europeana.eu/",
-                  :text => "Blog"
+                  url: 'http://blog.europeana.eu/',
+                  text: 'Blog'
                 },
                 {
-                  :url => root_url + "myeuropeana#login",
-                  :text => "My Europeana"
+                  url: root_url + 'myeuropeana#login',
+                  text: 'My Europeana'
                 }
               ]
             }  # end prim nav
           },
-          :footer =>{
-             :linklist1 =>{
-               :title => t('global.more-info'),
-               :items => [
+          footer: {
+             linklist1: {
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.